wp-content/plugins/authorizer/vendor/CAS-1.3.4/CAS/Client.php

```diff
@@ -314,10 +314,18 @@ private function _getServerBaseURL()
     {
         // the URL is build only when needed
         if ( empty($this->_server['base_url']) ) {
-            $this->_server['base_url'] = 'https://' . $this->_getServerHostname();
-            if ($this->_getServerPort()!=443) {
-                $this->_server['base_url'] .= ':'
-                .$this->_getServerPort();
+            // Sooo, to get this plugin working in a dev env, I had to change this method
```

Comment:
@StetsenTech @rshipp @hcli-work I just want to draw your attention to this change since I neglected to mention it in the PR description. Something to be aware of if we run "Update" on the Authorizer plugin in the Kits Wordpress install.

Reply:
does this actually make an http request if you're viewing kits with https, or does it just see that the port is 443 and use that correctly?

Reply:
> does this actually make an http request if you're viewing kits with https, or does it just see that the port is 443 and use that correctly?

It just sees the port and uses that correctly. If you're viewing the kits with https but have configured the port as something other than 443 or left it blank (this is in the plugin settings) then it will try to hit SSO over http.

Reply:
sounds risky, do we always have that port set to 443 in prod? don't want to send creds over http or something